genericjpa/README.md

```diff
@@ -0,0 +1,84 @@
+Hibernate-Search-JPA
+====================
```

Comment:
The async backend was originally created while building genericjpa.

The "async-updates" (asynchronous updates for Hibernate Search ORM) module is intended to be used in a context where at least the modules "util" (general utility classes nothing fancy), "db" (everything trigger related that is not orm specific) and "standalone" (standalone abstraction of the engine, you can already see that this is designed to be used by the actual genericjpa module, which will come later) are available.